addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16669,12 +16669,12 @@ msgctxt "#38103"
 msgid "Extract thumbnails from video files"
 msgstr ""
 
-#: system/settings/rbp.xml
+#: system/settings/settings.xml
 msgctxt "#38120"
-msgid "Support 8 channel DTS HD audio"
+msgid "Enable 8 channel DTS HD audio (CPU) decoding"
 msgstr ""
 
-#: system/settings/rbp.xml
+#: system/settings/settings.xml
 msgctxt "#38121"
-msgid "This option supports 8 channel DTS HD but may use more CPU "
+msgid "This option supports 8 channel DTS HD decoding on the CPU. This option is only available when DTS-HD audio decoding is disabled"
```

Comment:
Shouldn't this be: "This option is only available when DTS-HD audio **passthrough** is disabled"?